examples/rgb_slider/build.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
```

Comment:
Every new rust / .60 files will need a copyright header (we have a test for that)